system/jlib/jtrace.hpp

```diff
@@ -55,6 +55,19 @@ enum class SpanFlags : unsigned
 };
 BITMASK_ENUM(SpanFlags);
 
+static constexpr const char * NO_STATUS_MESSAGE = "";
+static constexpr int UNKNOWN_ERROR_CODE = -1;
+
+struct SpanError
```

Comment:
question: Does this provide enough value, rather than passing two parameters.  Was it to simplify optional parameters?  I'm happy for it to stay, just curious.